src/Disks/DiskEncrypted.cpp

```diff
@@ -29,6 +29,7 @@ namespace
 
     constexpr Algorithm DEFAULT_ENCRYPTION_ALGORITHM = Algorithm::AES_128_CTR;
 
+    template<typename String>
     String unhexKey(const String & hex)
```

Comment:
Why do we need to use a template here?

```suggestion
    NoDumpString unhexKey(const NoDumpString & hex)
```